README.md

```diff
@@ -7,7 +7,19 @@ Journal Club Updates
+11-02-2021 | *IBM’S Watson Analytics for Health Care: A Miracle made true* | Mayank et al | Geoffrey Kimani | [Paper](https://drive.google.com/file/d/1k_mnG8mahJdSvVbiikFVBL04LFTC6omc/view?usp=sharing) | [Presentation_link](https://drive.google.com/file/d/1d-D63oBLoszthrm3I5oI7vHZMq-pVaKa/view?usp=sharing)
+25-02-2021 | *A Brief history of bioinformatics* | Jeff et al | Ruth Nanjala | [Paper](https://academic.oup.com/bib/article-abstract/20/6/1981/5066445) | [Presentation_link](https://docs.google.com/presentation/d/1YlxyFsrnF67pShl5kbLioi-a29cse7SfFbxRTBVehQU/edit#slide=id.p)
+11-03-2021| *Three-gene PCR and high-resolution melting analysis for differentiating vertebrate species mitochondrial DNA for biodiversity research and complementing forensic surveillance*| Ouso et al | Winfred Gatua | [Paper](https://www.nature.com/articles/s41598-020-61600-3.pdf)|[Presentation link](https://drive.google.com/file/d/14SuZNBgU59rSdVB-e5BJe0CeUp-jyhE6/view?usp=sharing)
+25-03-2021 | *PhyResSE: a Web Tool Delineating Mycobacterium Tuberculosis Antibiotic Resistance and Lineage from Whole-Genome Sequencing Data* | Silke Feuerriegel et al | Rose Wambui | [Paper](https://jcm.asm.org/content/53/6/1908) | [Presentation_link](https://docs.google.com/presentation/d/1j11cCrxxwYnv98eQpwof5FhCqEJ1WOV_iwl7Rkv6JTk/edit#slide=id.p7)
+08-04-2021 | *Molecular Data storage using DNA* | Luis et al | Simeon Hebrew | [Paper](https://www.nature.com/articles/s41576-019-0125-3.epdf?sharing_token=d2W2PbBlMvR_zDhcnxYnQdRgN0jAjWel9jnR3ZoTv0PVUBRMJlc9VcYsdFg2SNetpyPZNEaTJF-YYhxMBj3YJ9hsPUHLHfMPvqwixiHFXfgqWmYTMZu_wOQcu8DV_n3M1Q083Vyhy1Z8S7_yylWO51MIS4qv9_13S-4Dyt_bDTFL7PRnU1O_DNDeHEmtdTsiPVLUh63ENd-T_KoTbJ_O0a0oHQvqO7ql8sPrkwbs2jg=&tracking_referrer=www.microsoft.com) | [Presentation_link](https://docs.google.com/presentation/d/1pMBGB_m2M6kyceQW60eQc0l02rblXX2vbzY-OjfK5gA/edit?usp=sharing)
+22-04-2021 | *Privacy challenges and research opportunities for genomic data sharing* | Bonomi et al | Nabwire Asatsa | [Paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7761157/) | [Presentation_link](https://docs.google.com/presentation/d/1qY60fNFnb2MWVMfmV6o7XoAdViXTzRn8FwaOxbvEY5g/edit#slide=id.p) 
+6-05-2021 | *COVID-19 vaccines: where we stand and challenges ahead.* | Forni et al | Kauthar Omar | [Paper](https://www.nature.com/articles/s41418-020-00720-9) | [Presentation link](https://docs.google.com/presentation/d/1vokp3xYIYHj1P6ToBcwoGCqVh8WYMY1WbDbvFIHQAv8/edit?usp=sharing)
+20-05-2021 | *TensorFlow: Biology’s Gateway to Deep Learning?* | Ladislav et al | Wilson Mudaki | [Paper](https://www.cell.com/cell-systems/fulltext/S2405-4712(16)00010-7) | [Presentation link](https://github.com/totodingi/Journal_Club/blob/main/presentations/present.pdf)
+
```

Comment:
03-06-2021 | *Unmapped exome reads implicate a role for Anelloviridae in childhood HIV-1 long-term non-progression* | S. Mwesigwa et al. | Kariuki Eric | [Paper](https://www.nature.com/articles/s41525-021-00185-w) | [Presentation_link](https://docs.google.com/presentation/d/1JIi0fToXyxqmYgqCx7a_JQL7whJnPIasbORyzB4UdvY/edit?usp=sharing)